bin.src/imsim.py

```diff
@@ -9,10 +9,9 @@
 import os
 import argparse
 import numpy as np
-from lsst.obs.lsstSim import LsstSimMapper
 from lsst.sims.coordUtils import chipNameFromRaDec
 from lsst.sims.GalSimInterface import SNRdocumentPSF
-from lsst.sims.GalSimInterface import LSSTCameraWrapper, GalSimCameraWrapper
```

Comment:
Is GalSimCameraWrapper begin removed because it was just superfluous?

Reply:
yes.